clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/restapiv2/ClusterControllerStateRestAPI.java

```diff
@@ -119,7 +94,7 @@ public SetResponse setUnitState(final SetUnitStateRequest request) throws StateR
         {
             @Override
             public Request<? extends SetResponse> visitCluster(Id.Cluster id) {
-                return new SetNodeStatesForClusterRequest(id, request);
+                return new SetNodeStatesForClusterRequest(request);
```

Comment:
Is the cluster ID redundant because this always happens in the context of a particular content cluster (i.e. the routing is done on "the outside") or is it later inferred from the request itself?

Reply:
Yes, the context decides the cluster